nautobot/utilities/tests/test_filters.py

```diff
@@ -207,50 +208,63 @@ def test_filter_null(self):
 
         kwargs = {"power_panels": [settings.FILTERS_NULL_CHOICE_VALUE]}
         qs = self.SiteFilterSet(kwargs, self.queryset).qs
+        expected_result = list(Site.objects.filter(powerpanel__isnull=True))
 
-        self.assertCountEqual(list(qs), [self.site0])
+        self.assertCountEqual(list(qs), expected_result)
```

Comment:
Can we change all of these `assertCountEqual` to use `assertQuerysetEqual` while you're in here? Not blocking.